README.md

```diff
@@ -58,6 +58,74 @@ async fn main() {
 }
 ```
 
+## Running on Lambda
+
+The Restate Rust SDK supports running services on AWS Lambda using Lambda Function URLs. This allows you to deploy your Restate services as serverless functions.
+
+### Setup
+
+First, enable the `lambda` feature in your `Cargo.toml`:
+
+```toml
+[dependencies]
+restate-sdk = { version = "0.1", features = ["lambda"] }
+tokio = { version = "1", features = ["full"] }
+```
+
+### Basic Lambda Service
+
+Here's how to create a simple Lambda service:
+
+```rust
+use restate_sdk::prelude::*;
+
+#[restate_sdk::service]
+trait Greeter {
+    async fn greet(name: String) -> HandlerResult<String>;
+}
+
+struct GreeterImpl;
+
+impl Greeter for GreeterImpl {
+    async fn greet(&self, _: Context<'_>, name: String) -> HandlerResult<String> {
+        Ok(format!("Greetings {name}"))
+    }
+}
+
+#[tokio::main]
+async fn main() {
+    // To enable logging/tracing
+    // tracing_subscriber::fmt::init();
+
+    // Build and run the Lambda endpoint
+    Endpoint::builder()
+        .bind(GreeterImpl.serve())
+        .build_lambda()
+        .run()
+        .await
+        .unwrap();
+}
+```
+
+### Deployment
+
+1. Install `cargo-lambda`
+   ```
+   cargo install cargo-lambda
+   ```
+2. Build your Lambda function:
+
+   ```bash
+   cargo lambda build --release --arm64 --output-format zip
+   ```
+
+3. Create a Lambda function with the following configuration:
+
+   - **Runtime**: Amazon Linux 2023
+   - **Architecture**: arm64
+
+4. Upload your `zip` file to the Lambda function.
```

Comment:
Can you add these as module docs, similar to how it was done in `http_server.rs`? The rustdocs we have are the main docs for the rust sdk (we don't have additional docs on docs.restate.dev), so we need to make these as complete as possible.

You can also check how it looks by running `cargo doc --open -p restate_sdk`